project_euler/problem_80/sol1.py

```diff
@@ -0,0 +1,31 @@
+"""
+    Euler Problem : 80
+    @author : Sandeep Gupta
+    @time   : 5 October 2020, 18:30
+    @Solution: Used decimal python module to calculate the decimal
+    places up to 100, the most important thing would be take calculate
+    a few extra places for decimal otherwise there will be rounding
+    error.
+    @answer     : 40886
+"""
+import decimal
+
+
+def solution() -> int:
```

Comment:
Please include `doctest` and function `docstring` explaining in brief what this function does.

Reply:
Apologies, updated as per all the comments.